src/servers_non_y2/ag_udev_persistent

```diff
@@ -20,6 +20,7 @@
 # find current contact information at www.suse.com.
 
 package ag_udev_persistent;
+BEGIN { $ENV{Y2DISABLELANGUAGEPLUGINS} = "1"; }
```

Comment:
NP: Comment in code explaining WHY it is needed, would be nice.

Reply:
@jreidinger good point, thanks, added